assets/src/apps/delivery/layouts/deck/DeckLayoutHeader.tsx

```diff
@@ -107,16 +112,18 @@ const DeckLayoutHeader: React.FC<DeckLayoutHeaderProps> = ({
 
               <div className="name">{userName}</div>
               <div className={`score ${!showScore ? 'displayNone' : ''}`}>{scoreText}</div>
-              <button
-                className="optionsToggle"
-                title="Toggle menu visibility"
-                aria-label="Toggle menu visibility"
-                onClick={() => {
-                  setShowOptions(!showOptions);
-                }}
-              >
-                <div className="icon-reorder"></div>
-              </button>
+              {!isReviewMode && (
```

Comment:
We don't want to show the top right menu that opens the history.  Review mode has it's own navigation on top.